pkgs/development/interpreters/octave/default.nix

```diff
@@ -157,6 +161,7 @@ stdenv.mkDerivation (finalAttrs: {
     ];
   nativeBuildInputs =
     [
+      perl
```

Comment:
Hmm maybe `perl` should be removed from `buildInputs`? I'm building `octave` now locally with this patch, `strictDeps` enabled and with `perl` removed from `buildInputs` and it passed the `configurePhase`...

Reply:
Just checked my native and cross build outputs, they do not depend on perl. Removed from buildInputs.